### ISSUE TITLE
Automatically focus new channels on join

### ISSUE THREAD

**Issue by RX14** [MCMrARM/revolution-irc#21]
When znc is replaying my backlog, its much easier to see when the backlog replay ends when a JOIN message automatically focuses the newly created channel tab. Also useful for noticing new PMs.

**Comment by Mikaela** [comment]
I think this should be an option as it would make my Revolution IRC usage more difficult as I generally `/znc detach *` and if I understood correctly, this change would make my client always jump to whatever buffer that suddenly gets attached by autoattach/message.

**Comment by RX14** [comment]
Yeah, I agree it should be an option. Personally i'd want autoattach to jump me to the new buffer but I can understand why you would not.

**Comment by Mikaela** [comment]
> Personally i'd want autoattach to jump me to the new buffer

Even if you were in the middle of typing something into another channel?

**Comment by RX14** [comment]
Well at worst it'd close the keyboard and be a minor annoyance. Although i guess you could cancel the change if the keyboard is up.

**Comment by MCMrARM** [comment]
I definitely wouldn't want this to be a feature or to be a default setting, but I may consider adding it as an option. For 0.4.0 I plan to make /join actually open the channel post join and that's why I am also assigning this issue the [bug] label and adding it to the 0.4.0 milestone. However, I currently have no plans to implement the auto-tab opening feature, but you are free to make a pull request for it.

**Comment by MCMrARM** [comment]
That part was fixed by https://github.com/MCMrARM/revolution-irc/commit/059448e394b934e374850d3eea571573fd4f86ce

**Comment by kurahaupo** [comment]
@MCMrARM Is this issue considered "fixed" yet?

Also useful would be (an option) to focus on a tab that's newly created in response to received data rather than an explicit /join or /msg, such as system notices or PM's.

I suspect that hiding the keyboard would be essential to this being useful rather than annoying.
